### DISCUSSION TITLE
Replace Webpack with a fast bundler

### DISCUSSION THREAD

**Discussion by itsmeadarsh2008** [reflex-dev/reflex#3030] [Ideas]
![image](https://github.com/reflex-dev/reflex/assets/71959210/4b868c0d-41d2-49cb-9d9c-8e96bc204a0a)

**Comment by Alek99** [comment]
Thanks for the suggestion, I haven't worked with esbuild before is it an easy switch for nextjs apps?

**Reply by Alek99** [reply]
If anyone is interested in taking this let me know.

**Reply by itsmeadarsh2008** [reply]
It can reduce the time for hot reload in `reflex run`
However, the parent company behind Next.js has also announced `turbopack` as the successor to `webpack`
I think it would be better if we switch to `turbopack` because of Next.js Ecosystem.

**Reply by Alek99** [reply]
Turbo pack is only for dev mode I believe wouldn't solve the production bundle time

**Comment by itsmeadarsh2008** [comment]
It's been a long time now, Any updates about changing the bundler?

**Comment by itsmeadarsh2008** [comment]
**References**: https://www.zaynetro.com/post/esbuild-project-setup with NextJS

**Comment by itsmeadarsh2008** [comment]
Updates?
